.travis.yml

```diff
@@ -2,6 +2,8 @@
 # RVM/bundler/ruby and whatnot. Right now 'rust' as a language actually
 # downloads a rust/cargo snapshot, which we don't really want for building rust.
 language: c
+compiler: /usr/bin/gcc-4.7
```

Comment:
another nit: can the treatment of CC and CXX be identical? might require setting the language to c++